### PR TITLE
Removes the beefmen trauma preferences, lightly reworks the way Sleepless Dreamer works

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

Since what to do with beefmen's choice of trauma has been a topic of discussion for some time, here's my latest idea for it:

- Beefmen no longer have a choice of trauma in their preferences. They will spawn with the Sleepless Dreamer trauma and **nothing else.**
- Using phobetor tears will have a chance to cause minor negative effects based on the user's sanity. The lower it is, the worse and more likely the effects will be, and they are now the primary source of hallucinations for beefmen, as well as giving mood debuffs. Examples below: 
<details>

<summary> Neutral to Good </summary>

https://github.com/fulpstation/fulpstation/assets/80640114/c7e99741-45e0-4a59-950e-42df6bd5bdbe

</details>

<details>

<summary> Neutral to Good </summary>

https://github.com/fulpstation/fulpstation/assets/80640114/9efe43f5-a2de-4276-baa6-4bdd3a291a10
</details>

<details>

<summary> Unstable to Good </summary>

https://github.com/fulpstation/fulpstation/assets/80640114/44fd5aee-9968-491b-aeb6-38af2b4b7f40

</details>

<details>

https://github.com/fulpstation/fulpstation/assets/80640114/20bb949d-0c79-4041-a5df-31d68581cb68

<summary> Crazy to Unstable </summary>

</details>

<details>

<summary> Insane to Crazy </summary>

https://github.com/fulpstation/fulpstation/assets/80640114/f32f5d78-d693-4621-8fe0-793a27a892d3

</details>

If you use one while within the Insanity to Crazy range you will also burn out your trauma, and be locked from using tears for 5 minutes.
## Why It's Good For The Game

The current selection of traumas is a bit questionable, since they range from annoying to very easily exploited to soft antag status (I'm looking at you, hypnotism).
I liked the idea of tying a beefman's insanity more closely to their use of the phobetor tears. If travelling through them is a power gained "after undergoing untold psychological hardship", it ought to feel like it. It's a good opportunity for flavour and to encourage managing your mood well, since sanity above great will prevent any downsides.
## Changelog
:cl:
del: Removed beefmen's brain trauma preference.
bal: Lightly reworks phobetor tears. They are now a source of hallucinations and mood debuffs that worsen the lower your sanity is while using them. Using one while crazy will also lock you out of using them for 5 minutes.
/:cl:

### CODE REVIEW

**Comment by remove32** [comment]
I dig this, but balance is questionable, lets do a TM to get some feedback on it?

**Comment by FernandoJ8** [comment]
> I dig this, but balance is questionable, lets do a TM to get some feedback on it?

Sure!

File: fulp_modules/features/species/beefmen/phobetor_trauma.dm [hunk 1]
```diff
@@ -74,6 +86,30 @@
 		return target
 	return FALSE
 
+/datum/brain_trauma/special/bluespace_prophet/phobetor/proc/burn_out(timer = 5 MINUTES)
+	burnt_out = TRUE
+
+	to_chat(owner, span_warning("Your feel yourself slipping back into the sleepless dream."))
```
Comment (Pepsilawn) [inline fulp_modules/features/species/beefmen/phobetor_trauma.dm:92]:
```suggestion
	to_chat(owner, span_warning("You feel yourself slipping back into the sleepless dream."))
```

File: fulp_modules/features/species/beefmen/phobetor_trauma.dm [hunk 2]
```diff
@@ -74,6 +86,30 @@
...
+	for(var/obj/effect/client_image_holder/phobetor/tear in created_firsts)
+		tear.deactivate()
+
+	if(timer)
+		addtimer(CALLBACK(src, PROC_REF(restore)), timer)
+
+/datum/brain_trauma/special/bluespace_prophet/phobetor/proc/restore()
+	burnt_out = FALSE
+
+	to_chat(owner, span_notice("Your eyes open once more, and with them unseen pathways."))
```
Comment (Pepsilawn) [inline fulp_modules/features/species/beefmen/phobetor_trauma.dm:103]:
```suggestion
	to_chat(owner, span_hypnophrase("Your eyes open once more, and with them unseen pathways."))
```

**Review by Pepsilawn — CHANGES_REQUESTED** [review]

File: fulp_modules/features/species/beefmen/phobetor_trauma.dm [hunk 3]
```diff
@@ -165,5 +219,88 @@
...
+			description = "The dreamscape seems gloomy today."
+		if(2)
+			description = "I've never liked the texture of the dream."
+		if(3)
+			description = "There's something offputting about the tears."
+
+/datum/mood_event/phobetor_major
+	description = "Something is stalking me through the tears."
+	mood_change = -6
+	timeout = 3 MINUTES
```
[the remark this answers is not in the document]
Comment (FernandoJ8) [reply fulp_modules/features/species/beefmen/phobetor_trauma.dm:277]: it's equivalent to being hungry

**Pull request merged**